src/components/templates/Search/Filters.tsx

```diff
@@ -27,33 +27,39 @@ export default function FilterPrice({
   serviceType,
   accessType,
   setServiceType,
-  setAccessType
+  setAccessType,
+  isSearch,
```

Comment:
Functionally it's working well. 
Some suggestions for improvement would be to give a more generic name to this `isSearch` prop like `addFiltersToUrl` or `showInUrl`, I think this would be a better name. 
And we can also omit the prop when not necessary, so remove `isSearch = {false}` when not needed

Reply:
makes sense will update this 👍